### ISSUE TITLE
Transparent Window Background Seemingly Opaque

### ISSUE THREAD

**Issue by D-Maxwell** [Vencord/Vesktop#433]
**Describe the bug**

Vesktop renders an opaque white sheet before all DOM elements, regardless of the Window Transparency setting.

**To Reproduce**

Turn on Window Transparency in the Settings.
Set the background of the page to any transparent colour.
Witness.

**Expected behavior**

I expected the theme I had been using for about a year over on BetterDiscord to render correctly on Vesktop ; it would seem that this is not the case.
Thinking that out of the many styling rules I had changed, perhaps only one would be truly problematic, I set out to debug through a simpler style.
Through QuickCSS --without any other skins enabled-- I set the `background-tertiary` variable to `transparent`.
Below stands a picture of the opaque oddity that occurred before my eyes.

**Screenshots**

![image](https://github.com/Vencord/Vesktop/assets/98100481/b747e773-cb99-4f7e-8d1e-c95c6707211c)

**Desktop (please complete the following information):**
 - OS/Distro: Windows 10
 - Version: 20H2

**Comment by Slimemaster0** [comment]
This is fixed in b636b65 how ever 1.5.0 came out 3 days prior to b636b65 being merged.
TLDR: Fixed in git but not in latest release.

**Comment by D-Maxwell** [comment]
Thank you both for the quick response. Looking forward to migrating fully to Vesktop.

**Comment by Slimemaster0** [comment]
> i will make a release later today

Yay thanks i have been wating for this!

**Comment by Vendicated** [comment]
Published in 1.5.1
